shell/platform/darwin/ios/framework/Source/FlutterEngineTest.mm

```diff
@@ -6,6 +6,15 @@
 #import <XCTest/XCTest.h>
 #import "flutter/shell/platform/darwin/ios/framework/Headers/FlutterEngine.h"
 
+#ifndef __has_feature
+#define __has_feature(x) 0 /* for non-clang compilers */
+#endif
+
+#if !__has_feature(objc_arc)
+#error ARC must be enabled!
```

Comment:
Out of curiosity what non-clang compilers are compiling these test files?

Reply:
None today that I know of.  If we accidentally used GCC it should probably be an error.  I'm just being cautious.  Ideally this would be a macro too, I just haven't gotten around to make it one.  It's not straightforward to get a macro of macros, I'd have to spend some time looking things up and fiddling with it.